server.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const db = require('./data/db');
+
+
+
+const port = 5000;
+const server = express();
+server.use(express.json());
+                 //homies
+server.get('/', (req, res) => {
+// 1st arg: route where a resource can be interacted with
+  // 2nd arg: callback to deal with sending responses, and handling incoming data.
+    res.send('Hello from express');
+});
+
+server.post('/api/users', (req, res) => {
+    const { name, bio } = req.body;
+    db
+        .insert({ name, bio })
+        .then(response => {
+            res.send(response);
+        })
+        .catch(error => {
+            res.json({ error });
+        });
+    //do something with
+});
+    server.get('/api/users', (req, res) =>{
+        db
+        .find()
+                .then(users => {
+                    res.json({ users });
+                })
+                .catch(error => {
+                    res.json({ error });
+                    //do something with error
+            });
```

Comment:
This set of closing punctuation -- `});` -- closes your `.catch` statement, but you also need closing punctuation (`})`) for your `server.get(/api/users'...` request.